docs/assemblies/proc_creating-a-set-of-data-plane-nodes.adoc

```diff
@@ -8,10 +8,21 @@ Create an `OpenStackDataPlaneNodeSet` CR for each logical grouping of nodes in y
 
 .Procedure
 
-. Copy the sample `OpenStackDataPlaneNodeSet` CR that you require and save it to a file named `openstack-edpm.yaml` on your workstation:
+. Copy the https://kustomize.io/[kustomize] examples on your workstation:
 
-* Pre-provisioned nodes: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml
-* Bare metal nodes: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml
+  git clone -n --depth=1 --filter=tree:0 \
+    https://github.com/openstack-k8s-operators/dataplane-operator
+  cd dataplane-operator
+  git sparse-checkout set --no-cone examples
+  git checkout
+
```

Comment:
`cd examples`

Reply:
They need to change directories first or run the `kustomize build` with `examples` in the path.

```
[fultonj@runcible test]$ git clone -n --depth=1 --filter=tree:0 https://github.com/fao89/dataplane-operator/ -b kustomize
Cloning into 'dataplane-operator'...
remote: Enumerating objects: 1, done.
remote: Counting objects: 100% (1/1), done.
remote: Total 1 (delta 0), reused 1 (delta 0), pack-reused 0
Receiving objects: 100% (1/1), done.
[fultonj@runcible test]$  cd dataplane-operator
[fultonj@runcible dataplane-operator{kustomize}]$ git sparse-checkout set --no-cone examples
[fultonj@runcible dataplane-operator{kustomize}]$ git checkout
remote: Enumerating objects: 64, done.
remote: Counting objects: 100% (64/64), done.
remote: Compressing objects: 100% (58/58), done.
remote: Total 64 (delta 0), reused 50 (delta 0), pack-reused 0
Receiving objects: 100% (64/64), 12.01 KiB | 12.01 MiB/s, done.
remote: Enumerating objects: 28, done.
remote: Counting objects: 100% (28/28), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 28 (delta 20), reused 26 (delta 20), pack-reused 0
Receiving objects: 100% (28/28), 8.56 KiB | 8.56 MiB/s, done.
Resolving deltas: 100% (20/20), done.
Your branch is up to date with 'origin/kustomize'.
[fultonj@runcible dataplane-operator{kustomize}]$ ls
examples
[fultonj@runcible dataplane-operator{kustomize}]$ kustomize build examples/preprovisioned/
apiVersion: dataplane.openstack.org/v1beta1
kind: OpenStackDataPlaneDeployment
...
```

Reply:
nice catch!